outline.md

```diff
@@ -42,8 +42,14 @@ Henry TODO
             3. Explanation of criteria and why each was selected - TODO for team: Fill these in
                 1. QKD criteria -
                 2. Confidentiality criteria -
-                3. Integrity criteria -
-                4. Non-Repudiation criteria
+                3. Integrity criteria 
+                    1. Data is Accurate
```

Comment:
I'm not seeing how 1-3 are relevant as they're just the definition of integrity.